$NetBSD: patch-configure,v 1.5 2020/06/20 19:03:37 adam Exp $

Make sure libpcsclite.so is not linked against -lphtread on NetBSD, as
loading libpthread.so through dlopen() is now forbidden

--- configure.orig	2014-09-24 13:04:46.000000000 +0000
+++ configure
@@ -680,6 +680,8 @@ LIBTOOL
 PTHREAD_CFLAGS
 PTHREAD_LIBS
 PTHREAD_CC
+LIBPCSC_PTHREAD_CFLAGS
+LIBPCSC_PTHREAD_LIBS
 ax_pthread_config
 ac_ct_AR
 AR
@@ -13732,7 +13734,7 @@ fi
 
 
 # disable libudev check is USB is not used
-if test "$use_usb" == "no"; then
+if test "$use_usb" = "no"; then
 	use_libudev="no"
 fi
 
@@ -13889,7 +13891,7 @@ fi
 
 
 # disable libusb check is USB is not used
-if test "$use_usb" == "no"; then
+if test "$use_usb" = "no"; then
 	use_libusb="no"
 fi
 
